test/static_fixtures/facts/facts_with_certname.json

```diff
@@ -89,6 +89,8 @@
     "netmask": "255.255.255.192",
     "fqdn": "sinn1636.lan.cert",
     "hostname": "sinn1636",
-    "domain": "lan.cert"
+    "domain": "lan.cert",
+    "foreman_uuid": "12345",
+    "smart_proxy_uuid": "23456"
```

Comment:
There is no signature, how this will work?

Oh I see down below, please keep in mind these tests are shared across other tests and this effectively renders it unusable for any other (and future) tests. I suggest to make sure the JSON is valid if possible (I understand there might be issues with oauth key). If this cannot be done, well, we have done our best.